mtp_noms_ops/apps/mtp_auth/forms.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from django import forms
+from django.conf import settings
+from django.utils.translation import gettext as _
+from mtp_common.user_admin.forms import SignUpForm as BaseSignUpForm
+from oauthlib.oauth2 import OAuth2Error
+from requests import RequestException
+from zendesk_tickets.forms import BaseTicketForm
+
+
+logger = logging.getLogger('mtp')
+
+
+class SignUpForm(BaseSignUpForm, BaseTicketForm):
+
+    account_request_zendesk_subject = _('MTP for digital team - Prisoner money intelligence - Request for new staff account')  # noqa: E501
+    zendesk_tags = ('feedback', 'mtp', 'noms-ops', 'account_request', settings.ENVIRONMENT)
+
+    role = forms.CharField(label=_('Role'), initial='security')
+    manager_email = forms.EmailField(label=_("Your manager's email"))
+
+    def user_already_requested_account(self):
+        try:
+            response = self.api_session.get(
+                'requests/',
+                params={
+                    'username': self.cleaned_data['username'],
+                    'role__name': self.cleaned_data['role'],
+                },
+            )
+            return response.json().get('count', 0) > 0
+        except (RequestException, OAuth2Error, ValueError):
+            logger.exception('Could not look up access requests')
+            self.add_error(None, _('This service is currently unavailable'))
+
+    def clean(self):
+        self.cleaned_data['role'] = 'security'
+
+        if self.is_valid() and self.user_already_requested_account():
+            # Subsequent account requests sent to ZenDesk are approved by Family Services
+            self.request.session['approver'] = _('Family Services')
```

Comment:
So this isn't actually true; it's FIU that review Intelligence Tool requests. But i'm not quite sure why you're adding this in the first place?

Reply:
FYI: [Showing always the same success message now](https://github.com/ministryofjustice/money-to-prisoners-noms-ops/pull/531/commits/391efe0259e2c1d7539652fc0c47196b4d098e9d) - resolving this.